uimodule/webapp/view/ListReport.view.xml

```diff
@@ -53,7 +53,7 @@
 
         <!-- DynamicPage Content -->
         <f:content>
-            <Table id="idProductsTable" 
+            <Table id="usersTableId" 
                 inset="false" 
```

Comment:
please check onFilterChange function and change table id in func too (71  line)

Reply:
mb you need to sync dev changes

Reply:
Will change after merge